requirements.txt

```
@@ -2,3 +2,4 @@
 pyyaml
 jinja2
 tabulate
+graphviz
```

```suggestion
graphviz
```
Something in your editor settings seems to strip off the last newline in each file.

---

Yes, you're right. I'll check for the new lines from next time. Thanks!